### PR TITLE
Traffic label transparent transmission plugin integration test demo: grpc demo, crossthread demo, util demo

### PR DESCRIPTION
**Pull request by daizhenyu**
【Fix issue】#1333

[Modification] Added a demo of the integration test part of the traffic label transparent transmission plug-in, including:
1.grpc-client-demo
2.grpc-server-demo
3.grpc-api-demo
4.crossthread-demo
5.tag-transmission-util-demo
Modified tryDownloadMidware.sh script file
Modified the action file of plugin-change-check and added the tag-transmission plugin
[Use case description] Integration test demo

[Self-test situation] 1. Local static check passed

[Scope of influence] None

### CODE REVIEW

File: sermant-integration-tests/tag-transmission-test/tag-transmission-util-demo/src/main/java/com/huaweicloud/demo/tagtransmission/util/HttpClientUtils.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+     */
+    public static String doHttpUrlConnectionGet(String url) {
+        String responseContext = null;
+        try {
+            URL serverUrl = new URL(url);
+            HttpURLConnection connection = (HttpURLConnection) serverUrl.openConnection();
+            connection.setRequestMethod("GET");
+
+            // 读取响应数据
+            BufferedReader in = new BufferedReader(new InputStreamReader(connection.getInputStream()));
```
Comment (hanbingleixue) [inline sermant-integration-tests/tag-transmission-test/tag-transmission-util-demo/src/main/java/com/huaweicloud/demo/tagtransmission/util/HttpClientUtils.java]: 出现异常时BufferedReader 可能无法关闭，建议在finally里面执行或者使用try-wth-resource

File: sermant-integration-tests/tag-transmission-test/tag-transmission-util-demo/src/main/java/com/huaweicloud/demo/tagtransmission/util/HttpClientUtils.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+     * jdkhttp get方法工具类
+     *
+     * @param url
+     * @return http请求的response
+     */
+    public static String doHttpUrlConnectionGet(String url) {
+        String responseContext = null;
+        try {
+            URL serverUrl = new URL(url);
+            HttpURLConnection connection = (HttpURLConnection) serverUrl.openConnection();
```
Comment (hanbingleixue) [inline sermant-integration-tests/tag-transmission-test/tag-transmission-util-demo/src/main/java/com/huaweicloud/demo/tagtransmission/util/HttpClientUtils.java]: 出现异常时HttpURLConnection可能无法关闭，建议在finally里面执行

File: sermant-integration-tests/tag-transmission-test/grpc-client-demo/src/main/java/com/huaweicloud/demo/tagtransmission/grpc/client/controller/GrpcClientController.java [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        DynamicMessage request = DynamicMessage.newBuilder(originMethodDescriptor.getInputType()).build();
+
+        // 使用 CompletableFuture 处理异步响应
+        CallOptions callOptions = CallOptions.DEFAULT;
+        CompletableFuture<DynamicMessage> responseFuture = new CompletableFuture<>();
+        ClientCalls.asyncUnaryCall(channel.newCall(methodDescriptor, callOptions), request,
+                new StreamObserver<DynamicMessage>() {
+                    @Override
+                    public void onNext(DynamicMessage value) {
+                        responseFuture.complete(value);
```
Comment (hanbingleixue) [inline sermant-integration-tests/tag-transmission-test/grpc-client-demo/src/main/java/com/huaweicloud/demo/tagtransmission/grpc/client/controller/GrpcClientController.java:100]: responseFuture.complete(value);是不是应该在下面的onCompleted()方法里面执行

Comment (daizhenyu) [reply sermant-integration-tests/tag-transmission-test/grpc-client-demo/src/main/java/com/huaweicloud/demo/tagtransmission/grpc/client/controller/GrpcClientController.java:100]: 应该不需要

File: sermant-integration-tests/tag-transmission-test/tag-transmission-util-demo/src/main/java/com/huaweicloud/demo/tagtransmission/util/HttpClientUtils.java (hunk 1 above)
Comment (daizhenyu) [reply sermant-integration-tests/tag-transmission-test/tag-transmission-util-demo/src/main/java/com/huaweicloud/demo/tagtransmission/util/HttpClientUtils.java]: 已修改

File: sermant-integration-tests/tag-transmission-test/tag-transmission-util-demo/src/main/java/com/huaweicloud/demo/tagtransmission/util/HttpClientUtils.java (hunk 2 above)
Comment (daizhenyu) [reply sermant-integration-tests/tag-transmission-test/tag-transmission-util-demo/src/main/java/com/huaweicloud/demo/tagtransmission/util/HttpClientUtils.java]: 已修改

File: sermant-integration-tests/tag-transmission-test/crossthread-demo/src/main/java/com/huaweicloud/demo/tagtransmission/crossthread/controller/CrossThreadController.java [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+    private static final int KEEP_ALIVE_TIME = 10;
+
+    private static final int QUEUE_CAPACITY = 20;
+
+    private static final int INITIAL_DELAY = 1;
+
+    private static final int DELAY = 10;
+
+    private static final int SLEEP = 5000;
```
Comment (lilai23) [inline sermant-integration-tests/tag-transmission-test/crossthread-demo/src/main/java/com/huaweicloud/demo/tagtransmission/crossthread/controller/CrossThreadController.java]: SLEEP_TIME

Comment (daizhenyu) [reply sermant-integration-tests/tag-transmission-test/crossthread-demo/src/main/java/com/huaweicloud/demo/tagtransmission/crossthread/controller/CrossThreadController.java]: 已修改

**Review by lilai23 — APPROVED** [review]

**Review by hanbingleixue — APPROVED** [review]

**Pull request merged**